config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -103,59 +104,7 @@ periodics:
     fork-per-release-replacements: "--extract=ci/latest -> --extract=ci/latest-{{.Version}}"
     testgrid-dashboards: google-windows, sig-windows-gce
     testgrid-tab-name: gce-windows-2019-master
-    description: Runs tests on a Kubernetes cluster with Windows 2019 nodes on GCE
-
-- name: ci-kubernetes-e2e-windows-gce-20h2
```

Comment:
Is there a specific reason to removing the job for gce-windows-20h2-master?

Reply:
That one was for dockershim.
I left in the 20h2-master-contaienrd job (right below this job definition).

Reply:
Oh, my bad. I forgot I had the other separate one for containerd. Makes sense. Thanks Mark for the update! LGTM